app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -244,45 +326,45 @@ You do not need to pay an agent or any other third party to complete your applic
 
 You will not be able to use eligibility for the IRP in your visa application as proof that you have enough money to support yourself.
 
-### Payment
+### Receiving payments
```

Comment:
This section is really lengthy - can it be broken up with sub-headings maybe?

It also jumps around from receiving the payment to needing a bank account to receiving the payment - is it worth reordering?

Reply:
Have reordered, not 100% sure I've done it right so will check after commits!